src/components/molecules/ChatsList/ChatsList.tsx

```diff
@@ -170,15 +171,32 @@ const ChatsList: React.FunctionComponent = () => {
                     setSelectedUserProfile={noopHandler}
                   />
                   <div className="private-message-content">
-                    <div className="private-message-author">{profileName}</div>
-                    <div className="private-message-last">
+                    <div
+                      className={`private-message-author ${
+                        isUnreadMessage && "unread"
+                      }`}
+                    >
+                      {profileName}
+                    </div>
+                    <div
+                      className={`private-message-last ${
+                        isUnreadMessage && "unread"
+                      }`}
+                    >
```

Comment:
We should add the `classnames` dependency, as it would allow us to make this sort of thing way clearer/more concise across the app.